pkg/sink/sink_test.go

```diff
@@ -970,7 +1135,7 @@ func TestHandleEvent(t *testing.T) {
 
 			// Sort TaskRuns when comparing (we do not know what order they were created in)
 			compareTaskRuns := func(x, y pipelinev1.TaskRun) bool {
-				return x.Name < y.Name
+				return x.Name < y.Name && x.Labels["triggers.tekton.dev/trigger"] < y.Labels["triggers.tekton.dev/trigger"]
```

Comment:
It seems like we are doing this because we are creating two TaskRuns with the same name. This is not an issue here since we are using FakeClients but for real use cases this will result in an error from the API server. 

We could probably turn `gitCloneTTSpec` into a simple function like `makeTTSpec(name, namespace string)` or something like that. If that is too much for this PR, would you mind opening an issue for this?